test/jdk/sun/security/provider/acvp/Launcher.java

```diff
@@ -37,12 +43,14 @@
 
 /// This test runs on `internalProjection.json`-style files generated
 /// by NIST's ACVP Server. See [https://github.com/usnistgov/ACVP-Server].
+/// Download zip archive, for instance -
+/// https://github.com/usnistgov/ACVP-Server/archive/refs/tags/v1.1.0.38.zip
+///
+/// The zip archive is either hosted on artifactory server or
+/// specified with local path to the test.
+/// The test looks for test data files in archive listed with `TEST_FILES`.
```

Comment:
"in archive" to "in the archive".

Now that you already have `TEST_FILES`, I think there is no need to use the `test.acvp.alg` test property anymore.

Also, to avoid confusing users, there’s no need to mention “sample files can be downloaded from…”. They can simply download the ZIP bundle.